src/cogs/trivia.py

```
@@ -112,6 +133,12 @@ def check(message) -> bool:
                     self.tally_scores(round_result)
                     await ctx.send(str(resp.author) + ' Wins!')
                     locked_channels.remove(ctx.channel.id)
+        else:
+            if ctx.channel.id not in avaible_channels:
+                return await ctx.send("This isn't a valid trivia channel!")
+
+            elif ctx.channel.id in locked_channels:
```

This case is probably a bad idea. It may disrupt the current game.

---

This still applies to line 155. Failing silently would be better.